src/model/base/FluxConditionPresentAssertion.js

```diff
@@ -150,6 +149,26 @@ class FluxConditionPresentAssertion extends FluxEntry {
         return undefined;
     }
 
+    // Returns the most recent toxicities within a specified time period
+    getMostRecentToxicities() {
+        // Set the max number of months prior to today that a toxicity can be
+        const numberOfMonths = 6; 
+        const sinceDate = moment().subtract(numberOfMonths, 'months');
+        let sinceDateMoment = new moment(sinceDate, "D MMM YYYY");
+
+        let tox = this.getToxicities();
+        let sortedTox = tox.sort(this._toxicitiesTimeSorter);
+        let mostRecentTox = [];
+
+        for (var i in sortedTox) {
```

Comment:
We probably should use `const` instead of `var`.  This can also probably be a `forEach` but you can stick with `for...in` if you prefer.

Reply:
👍

Reply:
Changed this to a forEach. Thanks Dan!